discourseparsing/rst_eval.py

```diff
@@ -1,41 +1,66 @@
-#!/usr/bin/env python3
-# License: MIT
+#!/usr/bin/env python
 
-from collections import Counter
+"""
+Script to evaluate the RST parser.
+
+:author: Michael Heilman
+:author: Nitin Madnani
+:organization: ETS
+"""
+
+import argparse
 import json
 import logging
+from collections import Counter
 from operator import itemgetter
 
 from nltk.tree import ParentedTree
 
-from discourseparsing.discourse_parsing import Parser
-from discourseparsing.discourse_segmentation import (Segmenter,
-                                                     extract_edus_tokens)
-from discourseparsing.parse_util import SyntaxParserWrapper
-from discourseparsing.rst_parse import segment_and_parse
-from discourseparsing.collapse_rst_labels import collapse_rst_labels
+from .collapse_rst_labels import collapse_rst_labels
+from .discourse_parsing import Parser
+from .discourse_segmentation import Segmenter, extract_edus_tokens
+from .parse_util import SyntaxParserWrapper
+from .rst_parse import segment_and_parse
 
 
 def _extract_spans(doc_id, edu_tokens_lists, tree):
-    # Precompute the token indices for each EDU.
+    """
+    Compute the token spans for each subtree in the RST tree.
+
+    Parameters
+    ----------
+    doc_id : dict
+        A dictionary representing the document.
+    edu_tokens_lists : list
+        List of list of tokens, one list for each EDU in the document.
+    tree : nltk.tree.ParentedTree
+        The given RST tree.
+
+    Returns
+    -------
+    spans : set
+        Set of (document_id, subtree label, span start index, span end index)
```

Comment:
is it `set of a () tuples` or just a `tuple`

Reply:
It's a set of tuples.

Reply:
Actually I'll double check.

Reply:
Yes, it's a set.